includes/wikia/DefaultSettings.php

```diff
@@ -1302,6 +1302,12 @@
 $wgAdDriverUseAdsAfterInfobox = false;
 
 /**
+ * @name $wgAdDriverUseGCS
+ * Whether to enable Google Consumer Surveys
+ */
+$wgAdDriverUseGCS = false;
```

Comment:
Would it be a big pain if we go with long name `$wgAdDriverUseGoogleConsumerSurveys`? :japanese_ogre: